packages/core/src/types/provider.ts

```diff
@@ -39,6 +39,12 @@ export interface LivepeerProvider {
   getAssetMetrics(args: GetAssetMetricsArgs): Promise<Metrics>;
 }
 
+export type WebhookRequest<TContext extends object> = {
```

Comment:
**nit:** Perhaps sth more specific like `PlaybackAccessControlRequest` would be better, WDYT? There are other kinds of webhooks on the API today and only the playback webhook has this request schema.